.github/workflows/ci.yml

```diff
@@ -54,3 +52,24 @@ jobs:
 
       - name: Run tests
         run: npm test --ignore-scripts
+
+  smoketest:
+    runs-on: ubuntu-latest
+    # if: ${{ github.actor != 'dependabot[bot]' }}
+    strategy:
+      matrix:
+        regex: []
+          # - "([0-9]|a[a-y])" # Starts with a number and goes up to `ay*`
+          # - "^az" # Starts with `az` (generally just Azure APIs)
+          # - "^([b-f]|g[^o]|[h-z])" # Goes from B to Z except if they start with `go`
+          # - "^go" # Starts with `go` (generally all Google APIs)
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - run: npm ci
+      - run: npm run build
+      - run: npm run test:smoke -- -- --regex="${{ matrix.regex }}"
```

Comment:
Is this correct with two sets of `--`?

Reply:
Unfortunately yeah. `npm run test:smoke` from the root runs `npm run test:smoke --workspace=packages/api` and I need to add the two `--` there to supply my data down into `smoketest.test.ts`. It's... not great.